src/pages/myTodo/tabs/ScrapPage.tsx

```
@@ -1,153 +1,59 @@
-        requiredEducationLevel: selectedJob.requiredEducationLevel,
-        closeType: '채용 시 마감',
-        salary: selectedJob.salary,
-        postTimestamp: '',
-        postDate: '',
-        'expiration-timestamp': '',
-        'expiration-date': selectedJob['expiration-date'],
-        deadline: selectedJob.deadline,
-        count: '0',
-      }
-    : null;
+  const jobs: any[] = [];
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)